test/functionalTest/cases/2604_op_query_attrs/op_query_attrs.test

```diff
@@ -33,10 +33,10 @@ brokerStart CB
 # 01. Create entity E1 with attributes A, B, C
```

Comment:
Rollback de change in this file done in PR https://github.com/telefonicaid/fiware-orion/pull/3751

NTC (informative)